docs/samples/tensorflow.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: "serving.kubeflow.org/v1alpha1"
+kind: "KFService"
+metadata:
+  name: "myModel"
+spec:
+  default:
+    tensorflow:
```

Comment:
do we need per mode type yaml? Can we standardize on schema and use that, with some values varying?

Reply:
We had a huge discussion on the merits of "default" and "canary" here: https://github.com/kubeflow/kfserving/issues/6

The high level is that this approach allows for a clean git diff. If you have additional suggestions, please write up your proposed spec and we'll take a look.

Alpha means we can still change the API :)